docs/docsite/rst/conf.py

```diff
@@ -334,15 +334,10 @@
 # with no 'notfound_template' and no 'notfound_context' set,
 # the extension builds 404.rst into a location-agnostic 404 page
 #
-# default is `en` - using this for the sub-site:
-notfound_default_language = "ansible"
-# default is `latest`:
 # setting explicitly - docsite serves up /ansible/latest/404.html
 # so keep this set to `latest` even on the `devel` branch
 # then no maintenance is needed when we branch a new stable_x.x
-notfound_default_version = "latest"
-# makes default setting explicit:
-notfound_no_urls_prefix = False
+notfound_urls_prefix = "/ansible/latest/"
```

Comment:
One of the related problems we have with the 404 page is https://github.com/ansible/ansible-documentation/issues/94 (TL;DR; all 404 pages have the /latest/ for left-hand navigation). I'm wondering if we update this value to match the branch, it would fix that older problem? I'm thinking this line goes to "ansible/devel/" in devel, and the stable-2.16 version goes to latest?  No impact on this PR but wondering if you think that would work.

Reply:
Yeah, that should be adjusted per branch. I'm not sure what's the best way to do this (without having to do it manually all the time)...